R/app-ui.R

```diff
@@ -14,6 +16,14 @@ golem_add_external_resources <- function() {
     golem::activate_js(),
     golem::favicon(),
     tags$script(src = "www/readCookie.js"),
-    tags$script(htmlwidgets::JS("setTimeout(function(){history.pushState({}, 'STOP-AD submission reviewer', window.location.pathname);},2000);"))
+    tags$script(htmlwidgets::JS("setTimeout(function(){history.pushState({}, 'STOP-AD submission reviewer', window.location.pathname);},2000);")),
+    
+    tags$script(HTML("
+      $(document).on('keydown', '#main-panel_section-overall_score', function (e) {
+        if (e.key === 'e' || e.key === 'E') {
+          e.preventDefault();
+        }
```

Comment:
Could you provide a rationale around adding this HTML block?

Reply:
Yes, so a numericInput in shiny allows the input of the character `e` for exponentials. The request for this ticket was to disallow that - i didn't see a native way to do that in Shiny, but this javascript effectively just disallows the input of `e` - that's the main rationale.

Reply:
Consider adding a comment to describe what this does